__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -76,15 +76,13 @@ const AllFields = (
         shouldValidateInStory?: boolean;
         showSubmitButtonInStory?: boolean;
         disabled?: boolean;
-        readOnly?: boolean;
```

Comment:
This was originally used for setting the readOnly prop on the field. LabeledField sets the readOnly prop on the field now so this isn't needed!